src/components/Login/Login.js

```diff
@@ -1,96 +1,114 @@
-import React, { Component } from 'react'
-import { login } from './../UserFunctions'
-import './Login.scss';
+import React, { Component } from "react";
+import { login } from "./../UserFunctions";
+import "./Login.scss";
 
 class Login extends Component {
   constructor() {
-    super()
+    super();
     this.state = {
-      email: '',
-      password: '',
-      errors: {}
-    }
+      email: "",
+      password: "",
+      errors: {},
+    };
 
-    this.onChange = this.onChange.bind(this)
-    this.onSubmit = this.onSubmit.bind(this)
+    this.onChange = this.onChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
   }
 
   onChange(e) {
-    this.setState({ [e.target.name]: e.target.value })
+    this.setState({ [e.target.name]: e.target.value });
   }
   onSubmit(e) {
-    e.preventDefault()
+    e.preventDefault();
 
     const user = {
       email: this.state.email,
-      password: this.state.password
-    }
+      password: this.state.password,
+    };
 
-    login(user).then(res => {
+    login(user).then((res) => {
       if (res) {
-        this.props.history.push(`/profile`)
+        this.props.history.push(`/profile`);
       }
-    })
+    });
   }
 
   render() {
     return (
       <div className="container-login">
-      <style>{'body { background: linear-gradient(180deg, #0F2D58 0%, #2C5797 55.91%); }'}</style>
-      <div className="row">
-      <div className="col-md-6 mt-5 mx-auto">
-      <h1 className="header" >Login to your account</h1>
-      <fieldset className="fieldset">
-      <form noValidate onSubmit={this.onSubmit}>
-      <div className="form-group mt-5 ml-5">
-      <label className="label required" htmlFor="email">Email</label>
-      <input
-      type="email"
-      className="form-control form-input input-sm"
-      name="email"
-      value={this.state.email}
-      onChange={this.onChange}
-      />
-      </div>
-      <div className="form-group ml-5">
-      <label className="label required mr-5" htmlFor="password">Password</label>
-      <label className="forgotpwd col-sm-7 ml-4 text-right">Forgot Password?</label>
-      <input
-      type="password"
-      className="form-control form-input input-sm"
-      name="password"
-      value={this.state.password}
-      onChange={this.onChange}
-      />
-      </div>
-      <div className="form-check ml-5">
-      <input type="checkbox" class="form-check-input" name="Rememberme" />
-      <label class="label mb-1" forHTML="Rememberme">Remember Me</label>
-      </div>
+        <style>
```

Comment:
Avoid inline styles. Add all styles in respective scss files. Will help keep all styles maintainable. Also should avoid styling entire body element.
Wrap element to style in a div and apply style only in that div.